### DISCUSSION TITLE
Monitoring via Grafana

### DISCUSSION THREAD

**Discussion by PPMagick** [wg-easy/wg-easy#1006] [Q&A]
There is a need to enable monitoring to track the number of connected peers and consumed traffic in Grafana, how can I do this? Run wg-easy in Docker, all boards (Wireguard + WireRest (JVM) ID dashboard 19458 ) require installation to the root, tell me how to configure this monitoring?
![image](https://github.com/wg-easy/wg-easy/assets/167945077/d3af9b13-9aba-477e-ad66-2f6e2ae86570)

**Comment by eandersons** [comment]
Right now two possible solutions come into mind:

- build a custom WireGuard Easy image which has installed and runs WireRest as well;
  that will definitely require some effort;
- [run WireGuard Easy without Docker](/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-without-Docker) and install WireRest on the host;
  obviously configuration files (`wg0.conf` and `wg0.json`) should be migrated to keep existing peers (clients).
